egs/wsj/s5/steps/nnet3/chain/train_more.py

```diff
@@ -0,0 +1,590 @@
+#!/usr/bin/env python
+
+# Copyright 2016    Vijayaditya Peddinti.
+#           2016    Vimal Manohar
+# Apache 2.0.
+
+""" This script is based on steps/nnet3/chain/train.sh
```

Comment:
Add in the description how this script is different from train.py and what it is for.